File: jsprobelms/day10/unq-ele.js
// print the unique elements in an array
let arr=[1,2,3,5,5,5,6,8,5,7,9,2,9,10]
res=[]
dup=[]
unq=[]
for (i of arr){
    if(!res.includes(i)){
        res.push(i)
    }
    else{
        dup.push(i)
    }
}
console.log(res,dup);
for(j of res){
    for(k of dup){
        if(!dup.includes(j)){
            unq.push(j)
            break
        }

    }
}
console.log(unq);

File: pythonproblems/day10/unq-ele.py
# print unique elements in array
arr=[1,2,4,5,1,2,4,5,8,7,9,6,3]
res=[]
dup=[]
unq=[]
for i in arr:
    if i not in res:
        res.append(i)
    else:
        dup.append(i)

for k in res:
   
    if k not in dup:
        unq.append(k)
print(unq)
